libgrive/src/json/Val.hh

```diff
@@ -138,6 +138,7 @@ template <> struct Val::Type2Enum<double>		{ static const TypeEnum type = double
 template <> struct Val::Type2Enum<std::string>	{ static const TypeEnum type = string_type ; } ;
 template <> struct Val::Type2Enum<Val::Array>	{ static const TypeEnum type = array_type ; } ;
 template <> struct Val::Type2Enum<Val::Object>	{ static const TypeEnum type = object_type ; } ;
+template <> struct Val::Type2Enum<std::vector<std::string> >	{ static const TypeEnum type = string_list_type ; } ;
```

Comment:
I did it this way because boost accepts program arguments with multiple values as a vector of strings, and I'm not quite sure how I could convert it to an array Val of string Vals